src/components/MyAccount/CheckoutsTab/RenewButton.tsx

```diff
@@ -28,22 +28,22 @@ const RenewButton = ({
     type: "default",
     bodyContent: (
       <Box className={styles.modalBody}>
-        <Text sx={{ marginLeft: "l" }}>
+        <Text>
           This item has been renewed. It is now due back on {checkout?.dueDate}.
         </Text>
       </Box>
     ),
     closeButtonLabel: "OK",
     headingText: (
       <Heading className={styles.modalHeading}>
-        <>
+        <Text>
```

Comment:
I don't think it's valid HTML to have a `p` element inside of an `h*` element. Can you move or add any necessary styles to the `Heading` parent?